src/explore-education-statistics-admin/src/pages/release/pre-release/components/PreReleaseUserAccessForm.tsx

```diff
@@ -81,28 +102,43 @@ const PreReleaseUserAccessForm = ({
         <Formik<FormValues>
           enableReinitialize
           initialValues={{
-            email: '',
+            emails: '',
           }}
           validationSchema={Yup.object<FormValues>({
-            email: Yup.string()
-              .required('Enter an email address')
-              .email('Enter a valid @education.gov.uk email address')
+            emails: Yup.string()
+              .trim()
+              .required('Please enter 1 or more email addresses')
+              .test({
+                name: 'number of lines',
+                message: `Please enter between 1 and ${inviteLimit} lines`,
+                test: (value: string) => {
+                  if (value) {
+                    const numOfLines = (value.match(/\n/g) || '').length + 1;
+                    return numOfLines <= inviteLimit;
+                  }
+                  return true;
+                },
+              })
               .test({
                 name: 'email format',
-                message: 'Enter a valid @education.gov.uk email address',
+                message: 'Please enter valid @education.gov.uk email addresses',
```

Comment:
No need to use 'please' - would suggest rewording this to 'Enter only @education.govuk email addresses' (or whatever the server-side error message is)